lib/src/main/java/my/nanihadesuka/compose/LazyColumnScrollbar.kt

```diff
@@ -74,225 +74,228 @@ fun LazyColumnScrollbar(
  */
 @Composable
 fun LazyColumnScrollbar(
-	listState: LazyListState,
-	rightSide: Boolean = true,
-	thickness: Dp = 6.dp,
-	padding: Dp = 8.dp,
-	thumbMinHeight: Float = 0.1f,
-	thumbColor: Color = Color(0xFF2A59B6),
-	thumbSelectedColor: Color = Color(0xFF5281CA),
-	thumbShape: Shape = CircleShape,
-	selectionMode: ScrollbarSelectionMode = ScrollbarSelectionMode.Thumb,
-	indicatorContent: (@Composable (index: Int, isThumbSelected: Boolean) -> Unit)? = null,
+    listState: LazyListState,
+    rightSide: Boolean = true,
+    thickness: Dp = 6.dp,
+    padding: Dp = 8.dp,
+    thumbMinHeight: Float = 0.1f,
+    thumbColor: Color = Color(0xFF2A59B6),
+    thumbSelectedColor: Color = Color(0xFF5281CA),
+    thumbShape: Shape = CircleShape,
+    selectionMode: ScrollbarSelectionMode = ScrollbarSelectionMode.Thumb,
+    indicatorContent: (@Composable (index: Int, isThumbSelected: Boolean) -> Unit)? = null,
 ) {
-	val coroutineScope = rememberCoroutineScope()
 
-	var isSelected by remember { mutableStateOf(false) }
+    val firstVisibleItemIndex =
+        remember { derivedStateOf { listState.firstVisibleItemIndex } }
```

Comment:
I don't see how this is going to improve recomposition count but anyway, its cool too. If you have time and really want to test performance, in the last android studio (electric eel) you can count the number of recompositions :=)